flixel/util/FlxStringUtil.hx

```diff
@@ -262,14 +262,8 @@ class FlxStringUtil
 	 */
 	public static function getDomain(url:String):String
```

Comment:
I guess the software engineer in me just couldn't leave well enough alone.  If you have the time, please take a look at the commit in my fork: seraku24/flixel@7937ffd.  There's absolutely no rush on this, but I figured I would get the work done while it was fresh in my head.  Do let me know if you think I should go ahead and submit the PR now.